### ISSUE TITLE
Configurable button.

### ISSUE THREAD

**Issue by outgaze** [JayRagon/triggerbotcpp#1]
I'm just suggesting it cause not everyone likes to use the t key maybe something like mouse 4 button would be better or a toggle :)

**Comment by JayRagon** [comment]
Yeah that would be good, but the triggerbot no longer works on valorant because they updated their anticheat to detect injected clicks the same way they detect injected mouse movement. I have been searching for bypasses and i will make an aimbot + triggerbot if it does work.
